.circleci/config.yml

```diff
@@ -118,6 +116,34 @@ jobs: # Integration Testing jobs
       - redmine-plugin/bundle-install
       - redmine-plugin/test:
           plugin: redmine_issues_tree
+  test-build-plugin-assets:
+    docker:
+      - image: circleci/node:12.18.2
```

Comment:
Could you add a parameter to specify node version?

Some of our projects depend on very old node.js 🙊

Reply:
This is just the integration test for the command itself

The node version for building the plugin assets should be specified via the `.node-version` file in the plugin folder (I actually added one for lgc recently because of that reason)